Wallets page styles

mahansky opened the pull request:
Fixes #1250 

Changes:
- hover effect on wallet action buttons (add address, rename..)
- opened wallet has lighter background 
- coins/hours are aligned to the right
- link to outputs is an icon instead of text 

Opened wallet:
![screen shot 2018-05-01 at 15 05 37](https://user-images.githubusercontent.com/1536298/39473848-32555d1a-4d51-11e8-8669-97d179248194.png)

Hover effect:
![screen shot 2018-05-01 at 15 05 58](https://user-images.githubusercontent.com/1536298/39473849-32763b02-4d51-11e8-9dcc-4e9373c674fc.png)

Senyoret1 commented:
That sounds good. The option to copy the address would be less accessible, but that should not be a big problem, hardly the users need to copy the addresses as fast as possible, and it will allow to easily add more options in the future, if necessary. Also, with that change it would be good to make the mouse pointer stay with the standard icon when it is over the rows, because now it seems to indicate that it is possible to click anywhere.

mahansky commented:
- lightened borders of opened wallet 
- removed `cursor:pointer` from address row
- fixed the alignment problem
- added small dropdown to wallet detail
- changed style of dropdowns (skycoin font + border radius)

Senyoret1 commented on src/gui/static/src/app/components/pages/wallets/wallet-detail/wallet-detail.component.html:
```diff
@@ -3,22 +3,25 @@
-      <p class="click-to-copy" [ngClass]="{ copying: address.copying }">
-        {{ address.address }} <span data-label="Copied" class="copy-label">Copy</span>
-      </p>
+      <p class="click-to-copy" [ngClass]="{ copying: address.copying }">{{ address.address }}</p>
```
`click-to-copy` and `copying` were removed from the css file.

Senyoret1 commented on src/gui/static/src/app/components/pages/wallets/wallet-detail/wallet-detail.component.html:
```diff
@@ -3,22 +3,25 @@
-                routerLink="/settings/outputs"
-                [queryParams]="{ addr: address.address }"
-      >chevron_right</mat-icon>
+      <mat-icon class="outputs-label" [matMenuTriggerFor]="optionsMenu">more_vert</mat-icon>
```
This is no longer a label, so the name of the css class should change.
